templates/adherent_profile/activity/_instances.html.twig

```diff
@@ -27,3 +27,33 @@
             </div>
     </article>
 </div>
+
+{% set pcMembership = app.user.politicalCommitteeMembership %}
+{% if pcMembership and pcMembership.politicalCommittee.isActive %}
+    <div class="l__col">
```

Comment:
what about checking `is_granted(COPOL_MEMBER_PERMISSION)` instead of 2 checks `pcMembership and pcMembership.politicalCommittee.isActive`?